internal/host/refresh_status_preprocessor.go

```diff
@@ -96,28 +96,30 @@ func (r *refreshPreprocessor) preprocess(c *validationContext) (map[string]bool,
 		conditions[cn.id.String()] = cn.fn(c)
 	}
 
-	// Validate operators
-	results, err := r.operatorsApi.ValidateHost(context.TODO(), c.cluster, c.host)
-	if err != nil {
-		return nil, nil, err
-	}
-	for _, result := range results {
-		id := validationID(result.ValidationId)
-		conditions[id.String()] = result.Status == api.Success
-		category, err := id.category()
+	if c.infraEnv == nil {
+		// Validate operators
+		results, err := r.operatorsApi.ValidateHost(context.TODO(), c.cluster, c.host)
 		if err != nil {
-			logrus.WithError(err).Warn("id.category()")
```

Comment:
Any reason to remove the error ?

Reply:
The log is not removed, it is just presented so by the diff. You can see on line 110

Reply:
the error is not removed, it is just how diff presents it, you can see it on line 110

Reply:
OK